.github/CODEOWNERS

```diff
@@ -635,7 +635,6 @@ embed.go @grafana/grafana-as-code
 /.github/workflows/issue-opened.yml @grafana/grafana-community-support
 /.github/workflows/metrics-collector.yml @torkelo
 /.github/workflows/milestone.yml @marefr
-/.github/workflows/ox-code-coverage.yml @grafana/explore-squad
```

Comment:
removed since not used by explore anymore... see https://github.com/grafana/code-coverage/pull/29#issuecomment-1788612302

Reply:
The @grafana/observability-logs was using that workflow to report coverage changes btw.